### DISCUSSION TITLE
Does html work in bios

### DISCUSSION THREAD

**Discussion by Shluffy** [theforumhelpers/theforumhelpers.github.io#228] [Q&A] [answered]
i tried [https://github.com/Shluffy/tfh-fork/blob/Bio-Test/forumhelpers/curators.json](https://github.com/Shluffy/tfh-fork/blob/Bio-Test/forumhelpers/curators.json) and [here](https://shluffy.github.io/tfh-fork/forumhelpers/) it won't upate

**Comment by Accio1** [comment, marked as answer]
It should. They are set using innerHTML so HTML tags should function correctly.

**Reply by Shluffy** [reply]
okay.

**Comment by leahcimto** [comment]
Also, the reason it doesn't update is because [here](https://github.com/theforumhelpers/theforumhelpers.github.io/blob/ba58074c5d09ea0f56f2c803aad9fabfe9856a90/forumhelpers/FHP.js#L3) you can see that it fetches the JSON from this site directly, not with a relative link.

**Reply by Shluffy** [reply]
true. I am going to change that.

**Reply by aidan-crockett** [reply]
Don't - as I've said before, your browser may block a CORS request to your own file system.
Fetching with an absolute url is the best way for it to work

**Reply by Accio1** [reply]
> Don't - as I've said before, your browser may block a CORS request to your own file system.
> Fetching with an absolute url is the best way for it to work

+1. If you want to make changes to those files, you have to have them hosted somewhere. I would recommend changing the URL to your own GitHub pages URL.

**Reply by Shluffy** [reply]
> > Don't - as I've said before, your browser may block a CORS request to your own file system.
> > Fetching with an absolute url is the best way for it to work
> 
> +1. If you want to make changes to those files, you have to have them hosted somewhere. I would recommend changing the URL to your own GitHub pages URL.

I did. I deleted the branch, but i didn't check. I realize now it was pulling non-existent data from my site rather then the right repo.  
@Accio1 @gosoccerboy5 @leahcimto thanks for the help. I am new to json

**Reply by aidan-crockett** [reply]
> > Don't - as I've said before, your browser may block a CORS request to your own file system.
> > Fetching with an absolute url is the best way for it to work
> 
> +1. If you want to make changes to those files, you have to have them hosted somewhere. I would recommend changing the URL to your own GitHub pages URL.

but then make sure to switch to the actual url before you merge the pr lol

**Reply by Shluffy** [reply]
i use a fork on a different branch. I use tfh-fork or tfh-clone
